src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/filter/index.tsx

```diff
@@ -60,28 +58,38 @@ function Filter({options, onFilter, className}: Props) {
           />
         )}
       >
-        <Content>
-          {Object.keys(options).map(category => (
-            <React.Fragment key={category}>
-              <Header>{category}</Header>
-              <List>
-                {options[category].map(groupedOption => {
-                  const {symbol, isChecked, id} = groupedOption;
-                  return (
-                    <StyledListItem
-                      key={id}
-                      onClick={handleClick(category, groupedOption)}
-                      isChecked={isChecked}
-                    >
-                      {symbol}
-                      <CheckboxFancy isChecked={isChecked} />
-                    </StyledListItem>
-                  );
-                })}
-              </List>
-            </React.Fragment>
-          ))}
-        </Content>
+        {({getMenuProps, isOpen}) => (
+          <StyledContent
+            {...getMenuProps()}
+            alignMenu="left"
+            width="240px"
+            isOpen={isOpen}
+            className="drop-down-filter-menu"
```

Comment:
Is this className used?

Reply:
yes 😉

Reply:
here https://github.com/getsentry/sentry/blob/218e6e036092b2f4cdb3708820c5e743df4c2617/src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidates.tsx#L394